check_diff/src/lib.rs

```diff
@@ -18,7 +18,7 @@ pub enum CheckDiffError {
     IO(std::io::Error),
 }
 
-pub struct GlobalRunners {
+pub struct CheckDiffRunners {
```

Comment:
`GlobalRunners` is still referred to within the file. You need to update all occurrences.